netsim/ansible/templates/initial/cumulus_nvue.j2

```diff
@@ -2,9 +2,9 @@
       {{ i.ifname }}:
         type: {{ 'svi' if i.type=='svi' else 'bond' if i.type=='lag' else 'swp' }}
         link:
-{%  if i.mtu is defined %}
+{%    if i.mtu is defined %}
```

Comment:
Seems like an indentation error to me

Reply:
See https://github.com/ipspace/netlab/pull/1557

Reply:
I have to admit my understanding of a "ready to merge PR" differs significantly from your "input for discussion" perspective.

This PR claims it implements multiple loopbacks on Cumulus NVUE, so let's focus on that.